.circleci/config.yml

```diff
@@ -2,6 +2,7 @@
 version: 2.1
 orbs:
   python: circleci/python@1.4.0
+  docker: circleci/docker@2.2
```

Comment:
Is there something that specifically needs `docker/machine`?  Seems like it would work with any executor.

Also this version is a couple years old so it might be better to use 2.6

Reply:
`ssh-keyscan -t rsa github.com >> ~/.ssh/known_hosts` didn't work with the `python/default` executor, so I copied the one we are using in bigquery-etl